src/views/user/ProfileUserView.jsx

```diff
@@ -117,4 +116,4 @@ class MyProfile extends Component {
     }
 }
 
-export default MyProfile
+export default ProfileUserView;
```

Comment:
Can't you just export the class at the top. So

export default class ProfileUserView extends Component